admin/javascript/src/config.js

```diff
@@ -26,8 +26,9 @@ class Config {
     const topLevelRoutes = [];
 
     Object.keys(window.ss.config.sections).forEach((key) => {
+      // Top level routes should have only one url-segment beneath admin/ and have a trailing slash
       const route = window.ss.config.sections[key].route;
-      const isTopLevelRoute = route.indexOf('/') === -1;
+      const isTopLevelRoute = route.match(/^admin\/[^\/]+\/?$/);
```

Comment:
Why does this distinction matter here in the first place? If a route is placed in `window.ss.config.sections[key].route`, isn't it always assumed to be "top level"? Other routes like GridField detail views won't be registered through that config key path.